pkg/storageos/rbac.go

```diff
@@ -38,11 +38,11 @@ const (
 	NFSClusterRoleName    = "storageos:nfs-provisioner"
 	NFSClusterBindingName = "storageos:nfs-provisioner"
 
-	SchedulerClusterRoleName    = "storageos:scheduler-extender"
-	SchedulerClusterBindingName = "storageos:scheduler-extender"
+	KubeSchedulerClusterRoleName    = "storageos:kube-scheduler"
+	KubeSchedulerClusterBindingName = "storageos:kube-scheduler"
```

Comment:
This name change is good but I'm afraid we shouldn't change an existing resource name. This will require migration process/step to the new version.